### PR TITLE
Proposed SR+CSM contracts

### PR DESCRIPTION
**Pull request by avsetsin**

### CODE REVIEW

File: docs/deployed-contracts/index.md [hunk 1]
```diff
@@ -8,13 +8,15 @@ pagination_next: deployed-contracts/holesky
...
 - Lido and stETH token: [`0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84`](https://etherscan.io/address/0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84) (proxy)
 - wstETH token: [`0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0`](https://etherscan.io/address/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0)
 - EIP-712 helper for stETH: [`0x8F73e4C2A6D852bb4ab2A45E6a9CF5715b3228B7`](https://etherscan.io/address/0x8F73e4C2A6D852bb4ab2A45E6a9CF5715b3228B7)
 - Staking Router: [`0xFdDf38947aFB03C621C71b06C9C70bce73f12999`](https://etherscan.io/address/0xFdDf38947aFB03C621C71b06C9C70bce73f12999) (proxy)
-- Node Operators registry: [`0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5`](https://etherscan.io/address/0x55032650b14df07b85bF18A3a3eC8E0Af2e028d5) (proxy)
-- Simple DVT: [`0xaE7B191A31f627b4eB1d4DaC64eaB9976995b433`](https://etherscan.io/address/0xaE7B191A31f627b4eB1d4DaC64eaB9976995b433) (proxy)
+- Staking Router: [`0xD8784e748f59Ba711fB5643191Ec3fAdD50Fb6df`](https://etherscan.io/address/0xD8784e748f59Ba711fB5643191Ec3fAdD50Fb6df) (impl)
+- (proposed) Staking Router: [`0x89eda99c0551d4320b56f82dde8df2f8d2ef81aa`](https://etherscan.io/address/0x89eda99c0551d4320b56f82dde8df2f8d2ef81aa) (impl)
```
Comment (bulbozaur) [inline docs/deployed-contracts/index.md:17]: It would be good to have link to forum of snapshot proposal for proposed contracts

Comment (avsetsin) [reply docs/deployed-contracts/index.md]: Thanks for the suggestion. Added links to the forum posts

File: docs/deployed-contracts/index.md [hunk 2]
```diff
@@ -47,11 +52,36 @@ pagination_next: deployed-contracts/holesky
 - Lido App Repo: [`0xF5Dc67E54FC96F993CD06073f71ca732C1E654B1`](https://etherscan.io/address/0xF5Dc67E54FC96F993CD06073f71ca732C1E654B1) (proxy)
 - Lido Oracle (Legacy Oracle) Repo: [`0xF9339DE629973c60c4d2b76749c81E6F40960E3A`](https://etherscan.io/address/0xF9339DE629973c60c4d2b76749c81E6F40960E3A) (proxy)
 - Node Operators Registry Repo: [`0x0D97E876ad14DB2b183CFeEB8aa1A5C788eB1831`](https://etherscan.io/address/0x0D97E876ad14DB2b183CFeEB8aa1A5C788eB1831) (proxy)
+- Simple DVT Repo: [`0x2325b0a607808dE42D918DB07F925FFcCfBb2968`](https://etherscan.io/address/0x2325b0a607808dE42D918DB07F925FFcCfBb2968) (proxy)
```
Comment (bulbozaur) [inline docs/deployed-contracts/index.md:55]: Why this address was deleted?

File: docs/deployed-contracts/index.md [hunk 3]
```diff
@@ -47,11 +52,36 @@ pagination_next: deployed-contracts/holesky
...
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSAccounting: [`0x4d72BFF1BeaC69925F8Bd12526a39BAAb069e5Da`](https://etherscan.io/address/0x4d72BFF1BeaC69925F8Bd12526a39BAAb069e5Da) (proxy)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSAccounting: [`0x71FCD2a6F38B644641B0F46c345Ea03Daabf2758`](https://etherscan.io/address/0x71FCD2a6F38B644641B0F46c345Ea03Daabf2758) (impl)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSFeeDistributor: [`0xD99CC66fEC647E68294C6477B40fC7E0F6F618D0`](https://etherscan.io/address/0xD99CC66fEC647E68294C6477B40fC7E0F6F618D0) (proxy)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSFeeDistributor: [`0x17Fc610ecbbAc3f99751b3B2aAc1bA2b22E444f0`](https://etherscan.io/address/0x17Fc610ecbbAc3f99751b3B2aAc1bA2b22E444f0) (impl)
+- ([proposed](https://research.lido.fi/t/community-staking-module/5917/67)) CSFeeOracle: [`0x4D4074628678Bd302921c20573EEa1ed38DdF7FB`](https://etherscan.io/address/0x4D4074628678Bd302921c20573EEa1ed38DdF7FB) (proxy)
```
Comment (bulbozaur) [inline docs/deployed-contracts/index.md:78]: Lets format CSFeeOracle proxy, implementation and consensus in the same way as all other protocol oracles
- CSFeeOracle:
   - CSFeeOracle: (proxy)
   - CSFeeOracle: (impl)
   - HashConsensus:

**Review by bulbozaur — APPROVED** [review]
LGTM

**Pull request merged**